mass_mailing_partner/__openerp__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     "name": "Link partners with mass-mailing",
-    "version": "8.0.2.0.0",
+    "version": "8.0.3.0.0",
```

Comment:
I changed a field name, which implies database change and module update, that's why I raised to 3.0.0.